docs/model-types.mdx

```diff
@@ -0,0 +1,467 @@
+
+### Validation Dataset
+
+The validation dataset is used during the validation phase of an ML model. It contains both feature variables and a target variable, like the training dataset. But as its name indicates, it is used to validate the predictions made by an ML model.
+
+The ML model takes only the feature variables from the validation dataset as input. Based on what the model learns during the training process, it makes predictions for the values of a target variable.
+
+Now comes the validation step. To assess the accuracy of the ML model, one compares the target variable values from the validation dataset with the target variable values predicted by the ML model. The closer these values are to each other, the better accuracy of the ML model.
+
+### Input Dataset
+
+After completing the training and validation phases, one can provide the input dataset consisting of only the feature variables to predict the target variable values.
+
+## How an ML Model is Created?
+
+In MindsDB, we use the [CREATE MODEL](/sql/create/predictor/) statement to create, train, and validate a model.
+
+### Training Phase
+
+Let's look at our training dataset. It contains both features and a target.
+
+```sql
+SELECT *
+FROM files.salary_dataset
+LIMIT 5;
+```
+
+On execution, we get:
+
+```sql
++---------+--------------+-----------+---------+--------+---------------+-------------------+------+
+|companyId|jobType       |degree     |major    |industry|yearsExperience|milesFromMetropolis|salary|
++---------+--------------+-----------+---------+--------+---------------+-------------------+------+
+|COMP37   |CFO           |MASTERS    |MATH     |HEALTH  |10             |83                 |130   |
+|COMP19   |CEO           |HIGH_SCHOOL|NONE     |WEB     |3              |73                 |101   |
+|COMP52   |VICE_PRESIDENT|DOCTORAL   |PHYSICS  |HEALTH  |10             |38                 |137   |
+|COMP38   |MANAGER       |DOCTORAL   |CHEMISTRY|AUTO    |8              |17                 |142   |
+|COMP7    |VICE_PRESIDENT|BACHELORS  |PHYSICS  |FINANCE |8              |16                 |163   |
++---------+--------------+-----------+---------+--------+---------------+-------------------+------+
+```
+
+Here, the features are `companyId`, `jobType`, `degree`, `major`, `industry`, `yearsExperience`, and `milesFromMetropolis`.
+
+And the target variable is `salary`.
+
+Let's create and train an ML model using this training dataset.
+
+```sql
+CREATE MODEL salary_predictor
+FROM files
+    (SELECT * FROM salary_dataset)
+PREDICT salary;
+```
+
+On execution, we get:
+
+```sql
+Query successfully completed
+```
+
+### Progress
+
+Here is how to check whether the training process is completed:
+
+```sql
+SELECT *
+FROM mindsdb.models
+WHERE name = 'salary_predictor';
+```
+
+Once the status is `complete`, the training phase is completed.
+
+### Validation Phase
+
+The `CREATE MODEL` statement performs validation of the model.
```

Comment:
Maybe change to:

> By default, ...

Reply:
Changed to _By default, the `CREATE MODEL` statement performs validation of the model._